packages/tldraw/src/lib/ui/context/actions.tsx

```diff
@@ -1366,6 +1366,60 @@ export function ActionsProvider({ overrides, children }: ActionsProviderProps) {
 					})
 				},
 			},
+			{
+				id: 'toggle-trackpad',
+				label: 'action.toggle-trackpad',
+				onSelect(source) {
```

Comment:
I think we should nix (!) these actions and supply the `onSelect` prop to the `TldrawUiMenuActionCheckboxItem` components instead of the `actionId` prop. should be a lot simpler.